ARTstor.js

```diff
@@ -1,131 +1,484 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
+	"translatorID": "7a690951-ed76-43f5-b761-707fef0345d4",
```

Comment:
no, please don't change the ID. That would leave the old translator in place and we'd have to delete that in a separate operation and there's no reason for that.